### ISSUE TITLE
Clearly a keylogger.

### ISSUE THREAD

**Issue by Packjackisback** [Eh3anKenZo/Token_Generator_discord#1]
Not only is the password protected zip archive and warning to turn off the antivirus extremely suspicious, but if you look in the code it's clearly a keylogger. You've got to at least hide it better.

**Comment by mvtbh** [comment]
yes also this guy has really bad trust already hacked like 50+ discord users and even he has a nsfw server
